src/main/java/core/basesyntax/Employee.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Employee {
+    private String name;
+    private int salary = 0;
+
+    public Employee(String name) {
+        this.name = name;
+    }
+
+    public void calcSalary(int hours, int rate) {
```

Comment:
This method should not be void. Or you need to rename the method. Calc salary means that you should return the value of calculation. Also, you are doing two operations here: calculating the new salary and increasing the existing value with the new salary value.

Reply:
done